### ISSUE TITLE
Create Automated Test Runner

### ISSUE THREAD

**Issue by 9radiopassive0** [BAlchemyLab/bal#3]
Using an automated test runner like [tox](https://tox.readthedocs.io/en/latest/) would be desirable at this stage. We can enforce PEP8 coding standards and coverage as well via it.

**Comment by sadov** [comment]
Yes, it would be great to make testing automation for our project. Another possibility -- using of [Travis CI service](https://travis-ci.org). As I understand, it have a good integration abilities with a GitHub.

**Comment by 9radiopassive0** [comment]
I will look into it once the setup.py issue is resolved.
